maquetado/setup/scss/tech.scss

```diff
@@ -0,0 +1,48 @@
+.tech-container {    
+  margin: 10px;  
+  height: 260px;
+  width: 200px;
+}
+
+.tech-back-container {
+  border: 2px solid  $green;
+  position: absolute;
+  border-radius: 10px;
+  width: 190px;  
+  height: 250px;
+}
+
+.tech-front-container {
+  background-color: $green;
+  position: absolute;
+  margin: 10px;
+  padding: 20px 5px;    
+  border-radius: 10px;
+  width: 190px;  
+  height: 250px;
+  display: flex;
+  flex-direction: column;  
+  align-items: center;  
+}
+
+.tech-img {
+  height: 100px;
+  width: 100px;
```

Comment:
This should work with a single attribute, height OR width, else the image can be bad resized